src/Engine/Graph/FileWriterNode.cpp

```diff
@@ -86,11 +87,10 @@ void FileWriterNode::Init()
 	attributeWriteMode->AddComboValue("Never Overwrite");
 	attributeWriteMode->AddComboValue("Always Overwrite");
 	attributeWriteMode->AddComboValue("Overwrite during Session");
-	attributeWriteMode->AddComboValue("Append");
```

Comment:
@JuliusCosmoRomeo @narek-h 
Have you tested what's happening if you open an existing classifier which has a FileWriter node configured to the now removed `Append` mode parameter? Is there an error/notification shown to the user when opening it? What value will be selected on the FileWriterNode instead? I guess it's ok to maybe silently switch those from the old `Append` to `Overwrite`? But at least Studio should not crash..